### DISCUSSION TITLE
Microsoft Visual C++ 2013 Redistributable (x64)  and (x86)

### DISCUSSION THREAD

**Discussion by kzeonG** [NexovaDev/UpdateHub#1322] [Help Thread]
With regards to the subject, i check gor updates and show both x86 & x64 need updating. After i click update, it says it couldn't update app. and then gives a button to download the file. I do this for both and then run each one and rebooting.
I then run UpdateHub and still shows the 2 need to be updated.
I know i can ignore but thought it best to advise as there may be an issue with your app that checks the version info or something.
anyway , just trying to help. Love the app by the way.
Is there a road map of features and whats next. tks

**Comment by rdeckard2019** [comment]
Check your installed programs. You most likely now have both the previous and the newer versions installed.
Not sure if this is actually a bug or not, since other update programs do this as well if both previous and current versions of an app are installed.
You could uninstall those older versions and the update notice should go away.
However, might be best to just ignore them if you're not sure they are required by any other apps on your machine.
